airflow/migrations/versions/0142_2_10_0_update_missing_constraints.py

```diff
@@ -0,0 +1,102 @@
+    with op.batch_alter_table('dag') as batch_op:
+        batch_op.alter_column('max_consecutive_failed_dag_runs', existing_type=sa.Integer(), nullable=False)
+
+    with op.batch_alter_table("task_instance") as batch_op:
+        batch_op.drop_constraint("task_instance_dag_run_fkey", type_="foreignkey")
+    with op.batch_alter_table("task_reschedule") as batch_op:
+        batch_op.drop_constraint("task_reschedule_dr_fkey", type_="foreignkey")
+
+    with op.batch_alter_table('dag_run') as batch_op:
+        batch_op.drop_constraint('dag_run_dag_id_execution_date_uq', type_='unique')
+        batch_op.drop_constraint('dag_run_dag_id_run_id_uq', type_='unique')
+        batch_op.create_unique_constraint('dag_run_dag_id_execution_date_key', ['dag_id', 'execution_date'])
+        batch_op.create_unique_constraint('dag_run_dag_id_run_id_key', ['dag_id', 'run_id'])
+    with op.batch_alter_table("task_instance") as batch_op:
+        batch_op.create_foreign_key("task_instance_dag_run_fkey", "dag_run", ["dag_id", "run_id"], ["dag_id", "run_id"])
+    with op.batch_alter_table("task_reschedule") as batch_op:
+        batch_op.create_foreign_key(
+            "task_reschedule_dr_fkey",
+            "dag_run",
+            ["dag_id", "run_id"],
+            ["dag_id", "run_id"],
+            ondelete="CASCADE",
+        )
+
+
+def downgrade():
+    """Unapply Update missing constraints"""
+    with op.batch_alter_table("task_instance") as batch_op:
+        batch_op.drop_constraint("task_instance_dag_run_fkey", type_="foreignkey")
+    with op.batch_alter_table("task_reschedule") as batch_op:
+        batch_op.drop_constraint("task_reschedule_dr_fkey", type_="foreignkey")
+    with op.batch_alter_table('dag_run') as batch_op:
+        batch_op.drop_constraint('dag_run_dag_id_run_id_key', type_='unique')
+        batch_op.drop_constraint('dag_run_dag_id_execution_date_key', type_='unique')
+        batch_op.create_unique_constraint('dag_run_dag_id_run_id_uq', ['dag_id', 'run_id'])
+        batch_op.create_unique_constraint('dag_run_dag_id_execution_date_uq', ['dag_id', 'execution_date'])
+
+    with op.batch_alter_table("task_instance") as batch_op:
+        batch_op.create_foreign_key("task_instance_dag_run_fkey", "dag_run", ["dag_id", "run_id"], ["dag_id", "run_id"])
+    with op.batch_alter_table("task_reschedule") as batch_op:
+        batch_op.create_foreign_key(
+            "task_reschedule_dr_fkey",
+            "dag_run",
+            ["dag_id", "run_id"],
+            ["dag_id", "run_id"],
+            ondelete="CASCADE",
+        )
+
+    with op.batch_alter_table('dag') as batch_op:
+        batch_op.alter_column('max_consecutive_failed_dag_runs',
+               existing_type=sa.Integer(),
+               nullable=True)
+
+    with op.batch_alter_table('connection') as batch_op:
+        batch_op.drop_constraint(batch_op.f('connection_conn_id_uq'), type_='unique')
+        batch_op.create_unique_constraint('unique_conn_id', ['conn_id'])
```

Comment:
Since we are just trying to make the migration and ORM path consistent, shouldn't these downgrades just do nothing?